crates/pallet-feeds/src/lib.rs

```diff
@@ -90,7 +73,7 @@ pub mod pallet {
 
             let feed_id = Self::current_feed_id();
 
-            Feeds::<T>::insert(feed_id, (H256::default(), u32::default()));
+            Feeds::<T>::insert(feed_id, Vec::<u8>::new());
```

Comment:
Why do you need to insert anything? `Feeds` already is `OptionQuery`, so it will return `None` if there wasn't anything there, isn't that acceptable? You cal also use non-option variant that will use `Vec::default()` by default. But I think it is indeed beneficial to know if it was an empty vector or `None`.

Reply:
`insert` requires two arguments

Reply:
What I'm saying is that don't need to do insert in the first place when creating a feed

Reply:
but I check if there is a key in the hashmap when we do `put`

Reply:
Then you probably don't need to, just check if `feed_id` is smaller than current. If it is, it exists.